examples/nl-core-FunctionalOrMentalStatus-01.xml

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Observation xmlns="http://hl7.org/fhir">
+   <id value="nl-core-FunctionalOrMentalStatus-01"/>
+   <meta>
+      <profile value="http://nictiz.nl/fhir/StructureDefinition/nl-core-FunctionalOrMentalStatus"/>
+   </meta>
+   <status value="final"/>
+   <category>
+      <coding>
+         <system value="http://snomed.info/sct"/>
+         <code value="118228005"/>
+         <display value="bevinding betreffende functioneren"/>
+      </coding>
+   </category>
+   <code>
+      <coding>
+         <system value="http://snomed.info/sct"/>
+         <code value="284888003"/>
+         <display value="kan zich scheren"/>
+      </coding>
+   </code>
+   <effectiveDateTime value="2015-02-23"/>
+   <valueCodeableConcept>
+      <coding>
+         <system value="http://snomed.info/sct"/>
+         <code value="284892005"/>
+         <display value="moeite met scheren"/>
+      </coding>
+   </valueCodeableConcept>
+   <note>
+      <text value="Dit is een toelichting"/>
```

Comment:
Could we include a more speaking comment, such as 'Door verminderde fijne motoriek niet mogelijk om het gezicht volledig glad te scheren'

Reply:
Will do! Thanks.